openmetadata-ui/src/main/resources/ui/src/components/Team/TeamDetails/TeamsHeaderSection/TeamsInfo.test.tsx

```diff
@@ -0,0 +1,140 @@
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+import { act, fireEvent, render, screen } from '@testing-library/react';
+import React from 'react';
+import { TeamType } from '../../../../generated/entity/teams/team';
+import { useAuth } from '../../../../hooks/authHooks';
+import { ENTITY_PERMISSIONS } from '../../../../mocks/Permissions.mock';
+import TeamsInfo from './TeamsInfo.component';
+
+const mockTeam = {
+  changeDescription: {},
+  children: [],
+  childrenCount: 0,
+  defaultRoles: [],
+  deleted: false,
+  description: 'Test team description',
+  displayName: 'Test Team',
+  domain: { id: 'test-domain', type: 'domain' },
+  email: 'test-team@test.com',
+  fullyQualifiedName: 'test-team',
+  href: '/test-team',
+  id: 'test-team',
+  inheritedRoles: [],
+  isJoinable: true,
+  name: 'test-team',
+  owner: { id: 'test-user', type: 'user' },
+  owns: [],
+  parents: [],
+  policies: [],
+  profile: {},
+  teamType: TeamType.Organization,
+  updatedAt: Date.now(),
+  updatedBy: 'test-user',
+  userCount: 1,
+  users: [{ id: 'test-user', type: 'user' }],
+  version: 1,
+};
+
+jest.mock('../../../../hooks/authHooks', () => ({
+  useAuth: jest.fn().mockReturnValue({ isAdminUser: true }),
+}));
+
+jest.mock('../../../common/OwnerLabel/OwnerLabel.component', () => ({
+  OwnerLabel: jest.fn().mockImplementation(() => <div>OwnerLabel</div>),
+}));
+
+jest.mock('../../../common/DomainLabel/DomainLabel.component', () => ({
+  DomainLabel: jest.fn().mockImplementation(() => <div>DomainLabel</div>),
+}));
+
+jest.mock('./TeamsSubscription.component', () => ({
+  __esModule: true,
+  default: jest.fn().mockImplementation(() => <div>TeamsSubscription</div>),
+}));
+
+jest.mock('../../../common/TeamTypeSelect/TeamTypeSelect.component', () => ({
+  __esModule: true,
+  default: jest.fn().mockImplementation(() => <div>TeamTypeSelect</div>),
+}));
+
+jest.mock('../../../Auth/AuthProviders/AuthProvider', () => ({
+  useAuthContext: jest.fn().mockReturnValue({
+    currentUser: { id: 'test-user' },
+  }),
+}));
+
+const mockEntityPermissions = { ...ENTITY_PERMISSIONS };
+
+const mockUpdateTeamHandler = jest.fn();
+const teamProps = {
+  parentTeams: [],
+  isGroupType: false,
+  childTeamsCount: 0,
+  currentTeam: mockTeam,
+  entityPermissions: mockEntityPermissions,
+  isTeamDeleted: false,
+  updateTeamHandler: mockUpdateTeamHandler,
+};
+
+describe('TeamsInfo', () => {
+  it('renders without crashing', () => {
+    const { container } = render(<TeamsInfo {...teamProps} />);
+    screen.debug(container);
```

Comment:
Please remove debug, also add some validation based on data testid for components to render chack